src/test/java/resources/validateChannel.feature

```diff
@@ -0,0 +1,52 @@
+Feature: Validate Channel Api Request Headers And Body
+
+  Scenario: Error Validity check for Post Transfer API for request body with no payee
+    Given I can create a TransactionChannelRequestDTO with no payee
+    When I call the post transfer API with expected status of 400
+    Then I should be able to assert the api validation for schema validation error response
+
+  Scenario: Unsupported header validation for Post Transfer Api Test
+    Given I can create a TransactionChannelRequestDTO
+    When I call the post transfer API having unsupported header with expected status of 400
+    Then I should get non empty response
+    Then I will assert that response body contains "error.msg.header.validation.errors"
+
+  Scenario: Required header validation for Post Transfer Api Test
+    Given I can create a TransactionChannelRequestDTO
+    When I call the post transfer API without required header Platform-TenantId with expected status of 400
+    Then I should get non empty response
+    Then I will assert that response body contains "error.msg.header.validation.errors"
+
+  Scenario: Error Validity check for Transaction Request API for request body with no payee
+    Given I can create a TransactionChannelRequestDTO with no payee
+    When I call the transaction request API with expected status of 400
+    Then I should be able to assert the api validation for schema validation error response
+
+  Scenario: Unsupported header validation for Transaction Request Api Test
+    Given I can create a TransactionChannelRequestDTO
+    When I call the transaction request API having unsupported header with expected status of 400
+    Then I should get non empty response
+    Then I will assert that response body contains "error.msg.header.validation.errors"
```

Comment:
please assert the actual error expected too

Reply:
asserted the actual error